### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by nikolaigut** [dimitri/pgloader#1526]
The last merge of a pull request was in October 2022, for this reason, my question: therefore this project still maintained by @dimitri or another contributor?

**Comment by svantevonerichsen6906** [comment]
I'd say, it is maintained in bursts, but with long pauses.

**Comment by aadrian** [comment]
@dimitri any news on this? There are quite a few unmerged PRs

**Comment by dimitri** [comment]
Yes, pgloader is actively maintained. **pgloader v4** (Clojure/JVM rewrite) is under active development in the `v4-clojure-rewrite` branch with a full GitHub Actions CI suite. Download the latest v4 JAR from the link below.

---
**pgloader v4** (Clojure/JVM, PR #1705):
```
curl -L -o pgloader.jar \
    https://github.com/dimitri/pgloader/releases/download/v4-dev/pgloader.jar
java -jar pgloader.jar --version
```
